src/advanced/node-management.md

```diff
@@ -415,6 +415,52 @@ A JSON object with the following fields:
 }
 ```
 
+### Consensus Enabled Info
+
+```none
+GET {system_base_path}/consensus_enabled
+```
+
+Returns a boolean value representing if the node participates in consensus.
+
+#### Parameters
+
+None.
+
+#### Response
+
+A JSON boolean.
+
+#### Response Example
+
+```JSON
+true
+```
+
+### Enabled/Disable Consensus Interaction
+
+```none
+POST {system_base_path}/consensus_enabled
+```
+
+Switches consensus interaction of the node on or off.
+
+#### Parameters
+
+- **enabled**: `bool`
+
+#### Example
+
+```None
+curl --data '{"enabled":false}' http://127.0.0.1:7780/api/system/v1/consensus_enabled
```

Comment:
Sorry to bug you again, but maybe add `-H "Content-Type: application/json"` to `curl` arguments just to be absolutely sure that the call does what we expect it to do? Afaik, without specifying the header it will be set to `application/x-www-form-urlencoded`.

Reply:
Done! :)